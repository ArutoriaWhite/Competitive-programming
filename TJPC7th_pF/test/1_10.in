54163392 2
16438979 1
25375374 1
