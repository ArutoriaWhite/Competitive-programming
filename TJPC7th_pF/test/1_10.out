16438979 8936395
